### ISSUE TITLE
Shutting down the server

### ISSUE THREAD

**Issue by jo** [hoodiehq/spawn-pouchdb-server#14]
It would be nice to be able to cleanly shut down the server, eg by exposing a `server.stop` method, [similar to hapi](http://hapijs.com/api#serverstopoptions-callback):

``` js
spawnPouchdbServer(options, function(error, server) {
  // ... finally:
  server.stop(function(error) {
    // server has been shut down
  })
})
```

**Comment by jo** [comment]
As I just discovered, an object with a `kill` method is [handed over to the callback](https://github.com/hoodiehq/spawn-pouchdb-server/blob/master/index.js#L72).

So we might just want to document this.

**Comment by gr2m** [comment]
one thought: wouldn't a reset method without killing the server be better?

**Comment by jo** [comment]
I like the idea! But we need a `stop` method anyway.

**Comment by gr2m** [comment]
agree, let's do it

**Comment by jo** [comment]
Its all there: https://github.com/hoodiehq/spawn-pouchdb-server/pull/16

**Comment by gr2m** [comment]
released as `spawn-pouchdb-server@3.2.0`
